addon/authenticators/torii.js

```diff
@@ -103,12 +103,24 @@ export default BaseAuthenticator.extend({
     @return {Ember.RSVP.Promise} A promise that when it resolves results in the session being invalidated
     @public
   */
-  invalidate() {
+  invalidate(data) {
+    this._assertToriiIsPresent();
+
+    data = data || {};
     return new RSVP.Promise((resolve, reject) => {
-      this.get('torii').close(this._provider).then(() => {
```

Comment:
For my API, yes, it is sufficient to just add `data` as a parameter to the method and then pass it as a second argument in the `close()` method. However, it's possible that some APIs may return data in the response that may be needed by the custom Torii provider. I don't know if that's a realistic expectation.